src/utils/webhooks.py

```
@@ -8,11 +8,26 @@
 
-import requests
+# ``requests`` is imported lazily via ``__getattr__`` to keep it off the
+# startup path (JTN-606).  Webhooks are only fired when a plugin refresh
+# fails, so importing the library at module load time is wasteful.
+# Existing tests that patch ``utils.webhooks.requests.post`` still work
+# because module ``__getattr__`` exposes the name on first attribute access.
 
 logger = logging.getLogger(__name__)
 
 
+def __getattr__(name: str) -> Any:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)